history/characters/48000_tirassian.txt

```diff
@@ -2716,19 +2716,22 @@
 	582.10.4={ death=yes }
 }
 48247={
-	name=Jamieson
+	name=Waylon # Presumably father of Lord Stormsong
 	dynasty=26002
 	dna="bmckgelfcbc"
 	culture=tirassian religion=tidemother
-	martial=7 diplomacy=5 stewardship=7 intrigue=5 learning=4
-	add_trait=intricate_webweaver add_trait=patient add_trait=arbitrary add_trait=ambitious 
-	add_trait=kind add_trait=proud 
+	martial=5 diplomacy=5 stewardship=7 intrigue=5 learning=6
+	add_trait=mastermind_theologian add_trait=erudite add_trait=patient add_trait=just add_trait=ambitious 
+	add_trait=diligent add_trait=kind add_trait=proud 
 	father=48243	#Barnard
-	547.8.2={ birth=yes add_trait=creature_human }
-	567.1.1={
-		add_trait=class_shaman_7
-	}
-	598.10.11={ death=yes }
```

Comment:
You changed date of death, but he still gives his titles to heir in 598.10.11, not in 575.04.16, so this happened...
![image](https://user-images.githubusercontent.com/46384992/57330292-a2784e00-7126-11e9-9f97-8d0326c3332e.png)

You need to edit these files...
![image](https://user-images.githubusercontent.com/46384992/57330343-c9368480-7126-11e9-9ba3-7d95933870ac.png)